proto/aserto/directory/reader/v3/reader.proto

```diff
@@ -265,17 +263,13 @@ message GetRelationsRequest {
     string subject_id                                           = 5     [(google.api.field_behavior) = OPTIONAL];
     // optional subject relation name
     string subject_relation                                     = 6     [(google.api.field_behavior) = OPTIONAL];
-    // materialize relation objects
-    bool with_objects                                           = 7     [(google.api.field_behavior) = OPTIONAL];
```

Comment:
Are we not going to have GetRelations with objects? I thought we agreed to keep it.